packages/connect-node/src/connect-protocol.ts

```diff
@@ -149,6 +152,35 @@ export function createConnectProtocol(
               ),
               responseTrailer: new Headers(),
             };
+
+            const timeout = connectParseTimeout(
+              requestHeader.get(connectTimeoutHeader) ?? ""
```

Comment:
No need to nullish coalesce to empty string:

```suggestion
              requestHeader.get(connectTimeoutHeader)
```